### PR TITLE
Update HOOMD LJ example to v3

### PR DESCRIPTION
**Pull request by joaander**
This PR requires https://github.com/mosdef-hub/mbuild/pull/871

### CODE REVIEW

File: hoomd_lj/multistate_hoomd_lj.ipynb
Comment (jennyfothergill) [inline hoomd_lj/multistate_hoomd_lj.ipynb:21]: <p>I think if you pin ipython to version 7.10.0, this message will be silenced.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mosdef-hub/mosdef-workflows/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (justinGilmer) [reply hoomd_lj/multistate_hoomd_lj.ipynb:21]: that is what i have found too. ill update the yml accordingly

Comment (joaander) [reply hoomd_lj/multistate_hoomd_lj.ipynb:21]: Should we disable warnings in the notebook? I would be concerned that pinning one particular dependency will break the environment file in the future (say when jupyterlab depends on a newer ipykernel)

Comment (justinGilmer) [reply hoomd_lj/multistate_hoomd_lj.ipynb:21]: What would you prefer in this situation? Id be ok with either option. It probably makes sense to filter out the deprecation warning instead.

If so, please feel free to undo my commit and we can merge, all seems well otherwise

Comment (joaander) [reply hoomd_lj/multistate_hoomd_lj.ipynb:21]: I don't have a strong preference.

Comment (justinGilmer) [reply hoomd_lj/multistate_hoomd_lj.ipynb:21]: Ok, lets instead filter the warning, unpin ipython, and when the ipykernel updates to the next major version, we can remove the filter _hopefully_

**Comment by joaander** [comment]
This is read to merge once https://github.com/mosdef-hub/mbuild/pull/871 is merged and there are new mbuild and HOOMD releases.

**Review by jennyfothergill — APPROVED** [review]
LGTM. The new hoomd3 schema is excellent. I was using this example to play around with https://github.com/mosdef-hub/mbuild/pull/871 

One note I have is it would be cool to add name, version, and combining rule to the ff.xml in this example to silence these warnings:
```
/Users/jenny/miniconda3/envs/mbuild-dev/lib/python3.8/site-packages/foyer/forcefield.py:615: UserWarning: No force field version number found in force field XML file.
  warnings.warn(
/Users/jenny/miniconda3/envs/mbuild-dev/lib/python3.8/site-packages/foyer/forcefield.py:627: UserWarning: No force field name found in force field XML file.
  warnings.warn(
/Users/jenny/miniconda3/envs/mbuild-dev/lib/python3.8/site-packages/foyer/forcefield.py:639: UserWarning: No combining rule found in force field XML file.
  warnings.warn(
```

also is wrap implemented with the snapshot box now too? We could maybe switch that as well.

**Comment by joaander** [comment]
> LGTM. The new hoomd3 schema is excellent. I was using this example to play around with [mosdef-hub/mbuild#871](https://github.com/mosdef-hub/mbuild/pull/871)
> 
> One note I have is it would be cool to add name, version, and combining rule to the ff.xml in this example to silence these warnings:
> 
> ```
> /Users/jenny/miniconda3/envs/mbuild-dev/lib/python3.8/site-packages/foyer/forcefield.py:615: UserWarning: No force field version number found in force field XML file.
>   warnings.warn(
> /Users/jenny/miniconda3/envs/mbuild-dev/lib/python3.8/site-packages/foyer/forcefield.py:627: UserWarning: No force field name found in force field XML file.
>   warnings.warn(
> /Users/jenny/miniconda3/envs/mbuild-dev/lib/python3.8/site-packages/foyer/forcefield.py:639: UserWarning: No combining rule found in force field XML file.
>   warnings.warn(
> ```

I am not familiar with the XML file format. Could someone add that?

> also is wrap implemented with the snapshot box now too? We could maybe switch that as well.

Yes, `Snapshot` can wrap particles: https://hoomd-blue.readthedocs.io/en/latest/package-hoomd.html#hoomd.Snapshot.wrap. This is used in `mbuild`: https://github.com/mosdef-hub/mbuild/blob/a81b4ebbca6b373076d0f63096cea8d91e9081f8/mbuild/formats/hoomd_snapshot.py#L361-L363

**Comment by joaander** [comment]
@justinGilmer any further changes needed before merging? I'd like to link to this from the HOOMD-blue documentation.

**Comment by jennyfothergill** [comment]
one change that still might be nice is to replace the 
```<Forcefield >``` 
line in the xml with 
```<Forcefield name="Argon-LJ" version="0.0.1" combining_rule="geometric">```
to silence the warnings. The name, version, and combining rule I suggest are all just off the top of my head so feel free to change them.

**Comment by justinGilmer** [comment]
Edits made to the XML, once we are happy with that, we can merge.

**Comment by justinGilmer** [comment]
@jennyfothergill re-requesting your review as well.

**Review by jennyfothergill — APPROVED** [review]
LGTM :)

**Comment by joaander** [comment]
Are these notebooks rendered anywhere? Or is the best case to link directly to the notebook in GitHub?https://github.com/mosdef-hub/mosdef-workflows/blob/master/hoomd_lj/multistate_hoomd_lj.ipynb

**Comment by jennyfothergill** [comment]
@joaander to my knowledge, no. But they definitely should be! I'll open an issue to add a docs page for this repo. It'd be nice to be able to link to it from mbuild/foyer/gmso docs.

**Comment by justinGilmer** [comment]
At the moment, we do not have any rendering on a website or elsewhere for these notebooks aside from GitHub's in-browser render. Although I am absolutely ok with us creating a page where we host these and then can link individual examples, etc to an html page. Lets not let this hold up this PR, I can merge this now.

**Review by justinGilmer — APPROVED** [review]

**Comment by joaander** [comment]
Sorry, I didn't mean to create extra work. I was just asking if there already was one.

**Comment by justinGilmer** [comment]
Not at all, I think it would be a very nice QOL improvement especially if we want more users to try these out.

**Pull request merged**
